pennylane/pulse/parametrized_hamiltonian.py

```diff
@@ -25,15 +25,12 @@
 class ParametrizedHamiltonian:
     r"""Callable object holding the information representing a parametrized Hamiltonian.
 
-    Passing parameters to the ``ParametrizedHamiltonian`` returns an
-    :class:`~pennylane.operation.Operator` representing the Hamiltonian for that set of parameters.
-
     The Hamiltonian can be represented as a linear combination of other operators, e.g.,
-    :math:`H(v, t) = H_\text{drift} + \sum_j f_j(v, t) H_j`, where the :math:`v` are trainable
-    parameters, and t is time.
 
-    For example, a time-dependent ``ParametrizedHamiltonian`` with a single trainable parameter can
-    be: :math:`a`, could be :math:`H = 2 X_1 X_2 + \sin(a t) Y_1 Y_2`
+    .. math::
+        H(\{v_j\}, t) = H_\text{drift} + \sum_j f_j(v_j, t) H_j
+
+    where the :math:`\{v_j\}` are trainable parameters for each scalar-valued parametrization :math:`f_j`, and t is time.
 
     Args:
         coeffs (Union[float, callable]): coefficients of the Hamiltonian expression, which may be
```

Comment:
You should probably include the fact that constant coefficients and their respective terms must come before parametrized ones. I noticed it on the pulse overview doc as well as in the usage details section for `ParametrizedHamiltonian`, but the usage details section is hidden by default on the PL docs so it's probably a good idea to include it here where it's going to be much more clear.